src/app/pages/OnDemandAudioPage/index.stories.tsx

```diff
@@ -22,7 +21,10 @@ const matchFixtures = service => ({
   },
 });
 
-const Component = (_, { service }) => {
+const Component = (
+  _: unknown,
```

Comment:
What does the underscore mean in this new and the old code?

Reply:
An underscore in js means "please ignore" i.e. an unused var. The alternative would be to give the parameter a name, and then add a linting override to permit the declaration of an unused var. 

It needs to be added because of the fact that this Component has positional parameters, param 1 being... something? not sure? and param 2 being an object (and here we're destructuring the service off that object)

Reply:
I think VSCode gives you the option to use underscore.

Say for example I did give the parameter a name e.g. param1, this is what I see in the "Quick Fix.." popup

![Screenshot 2024-09-03 at 13 46 40](https://github.com/user-attachments/assets/2c23ff22-e689-4276-b04c-21caac5a9db3)

Adding a preceding `_` to the variable is just the same as naming the variable `_`, and is slightly easier to skim over / ignore when you're reading the code.